config/locales/en.yml

```diff
@@ -395,6 +395,7 @@ en:
       Any changes you make to a live form will be updated in the form
       immediately.
     notification_title: Important
+  login_button: Login
```

Comment:
The GOV.UK style guide recommends using the phrase "sign in" to refer to logging in [[1](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#sign-in-or-log-in)]; this matches the language used on most services in DSP [[2](https://github.com/alphagov/govuk-design-system-backlog/issues/95#issuecomment-434643428)].

We should change this content, and also _ideally_ change the route and method names to be `sign-in` or `sign_in`.

Reply:
that makes sense, I'll change it.

It also reminds me that I need to change the sign-up link to use post too 🙄